pkg/chunk/chunk_store.go

```diff
@@ -132,7 +132,7 @@ func newStore(cfg StoreConfig, schema StoreSchema, index IndexClient, chunks Cli
 }
 
 // Stop any background goroutines (ie in the cache.)
-func (c *store) Stop() {
+func (c *baseStore) Stop() {
```

Comment:
Please move this function above, grouping it with `baseStore` after then `newBaseStore`